sycl/test-e2e/Basic/multi_ptr_null_relational_operators.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/sycl.hpp>
```

Comment:
How hard would it be to switch it to `<sycl/detail/core.hpp>` + something? I'd hope

```suggestion
#include <sycl/detail/core.hpp>
#include <sycl/multi_ptr.hpp>
```

should be enough.

Reply:
thanks @aelovikov-intel, done.